### PR TITLE
Manual for current node

### PR DESCRIPTION
**Pull request by munrocket**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,18 +34,14 @@ test('should result to the answer', t => {
...
 
 Moreover zora does not use specific platform API which should make it transparent to most of your tools such module bundlers or transpilers.
 
 In few words:
 > Zora is Ecmascript, no less, no more.
 
-### Node config
-
-In Node 12 you should add flags `--experimental-modules --es-module-specifier-resolution=node` and add `"type": "module"` to your project.json. In smaller versions use full path to zora library and .mjs extension in all your files.
-
```
Comment (lorenzofox3) [inline README.md]: or bundle your test program as commonjs

Comment (munrocket) [reply README.md]: cjs is not ES6, it’s specific markup for node. As I understand your library is unique, because it work without any bundler.

Comment (lorenzofox3) [reply README.md]: Yes. However in a usage note (by the way your paragraph should probably be moved to the usage section) you need to be pragmatic and think on how your users actually use the library. Assuming everybody uses node 12 is probably wrong. For example, in serverless environment the main providers still use node 10 if I am not wrong.

I see more something like:
usage
* Node
  * with ecmascript module syntax
    * Node < 12: bundler to cjs
    * Node 12: expermental flags
  * with cjs syntax: just run with node
* Browsers
  * es6 module
  * legacy browsers: bundler to iife

Something like that

Comment (munrocket) [reply README.md]: I am added your test runner here https://testanything.org/producers.html .
Please create an article on medium/dev.to

Comment (lorenzofox3) [reply README.md]: @munrocket 
[first article](https://dev.to/lorenzofox3/tools-and-the-design-of-a-testing-experience-2mdc) of an ongoing serie ! Thanks for the idea

Comment (munrocket) [reply README.md]: @lorenzofox3 example how to use zora with puppeteer and ES6 #47

**Comment by lillem4n** [comment]
I have no constructive stuff to add, except I was violently happy about this being written in modern ES6 modules. Woho! Lets hope node removes the need for that horrible flag soon, it should've been done in version 8.

**Comment by lorenzofox3** [comment]
@lillem4n Thanks. If you wish to run ESM module without any bundler, you can preload [esm](https://github.com/standard-things/esm). ``node -r esm some/ecmascript/file.js``

Regarding testing nodejs programs, I have released a [test runner](https://github.com/lorenzofox3/zora-node#readme) specifically for nodejs. Check it out and let me know what you think.

**Comment by munrocket** [comment]
Now zora added to this list: https://github.com/sindresorhus/awesome-tap
And current node 13.2 don't need any `--experimental-modules` flag

**Comment by lorenzofox3** [comment]
@munrocket thanks !

**Pull request closed without merging**